android/engine/src/main/java/org/smartregister/fhircore/engine/di/NetworkModule.kt

```diff
@@ -91,9 +91,9 @@ class NetworkModule {
   @Provides
   fun provideFhirResourceService(
     parser: IParser,
-    @OkHttpClientQualifier okHttpClient: OkHttpClient,
+    @AuthOkHttpClientQualifier okHttpClient: OkHttpClient,
```

Comment:
There was a reason for providing two different `OkHttpClient` instances, this was to limit the logging for different requests another was to use different interceptors to set the Bearer access token for particular requests. If those are no longer the intentions then perhaps we just need a single instance of the `OkHttpClient` thus the Qualifiers aren't necessary.

Reply:
Sure, we can use one instance for `OkHttpClient` as we need to have bearer access token for some calls in the `FhirResourceService` for fetching resources like `StructureMaps` and `Questionnaires`